### PR TITLE
ZKP tests for verifiable presentations

### PR DESCRIPTION
**Pull request by brentzundel**
Tests for verifiable presentations that support ZKPs.

### CODE REVIEW

**Comment by brentzundel** [comment]
@dmitrizagidulin @ken-ebert @dlongley @yancyribbens 
If any of you could review this PR, I would appreciate it.

File: test/vc-data-model-1.0/input/example-015-zkp-vp.jsonld [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "issuer": "did:example:Wz4eUg7SetGfaUVCn8U9d62oDYrUJLuUtcy619",
+    "issuanceDate": "2010-01-01T19:23:24Z",
+    "credentialSubject": {
+      "degreeType": "BachelorDegree",
+      "degreeSchool": "College of Engineering"
+    },
+    "proof": {
+      "type": "ex:AnonCredDerivedCredentialv1",
+      "primaryProof": "cg7wLNSi48K5qNyAVMwdYqVHSMv1Ur8i...Fg2ZvWF6zGvcSAsym2sgSk737",
+      "nonRevocationProof": "mu6fg24MfJPU1HvSXsf3ybzKARib4WxG...RSce53M6UwQCxYshCuS3d2h"
```
Comment (dlongley) [inline test/vc-data-model-1.0/input/example-015-zkp-vp.jsonld:26]: Any term (e.g. `primaryProof`, `nonRevocationProof`) should be defined in the core context or the example context. It seems like these are just example terms for testing, so we should add them to the example context. We can do the same with `AnonCredDerivedCredentialv1` in order to remove the `ex:` prefix if we want. The same goes for `degreeType` and `degreeSchool` used above. The alternative to adding new terms to the example context is to use whatever is already in there, if it applies to the use case.

cc: @dmitrizagidulin

**Comment by dmitrizagidulin** [comment]
👍 looks good.

@brentzundel - can you rebase this on latest? (to remove conflict)

@dlongley - agreed, I'm drawing up a list of terms to add to core and example contexts, from prs #38 and #39.

**Comment by brentzundel** [comment]
> @brentzundel - can you rebase this on latest? (to remove conflict)

I think I rebased it. let me know if I need to try again.

**Review by dmitrizagidulin — APPROVED** [review]
👍 should be ready to merge.

(I gathered new terms from this PR and added them to the context here: https://github.com/w3c/vc-data-model/pull/661 )

**Pull request merged**